packages/edit-site/src/components/error-boundary/warning.js

```diff
@@ -9,12 +9,7 @@ import { useCopyToClipboard } from '@wordpress/compose';
 function CopyButton( { text, children } ) {
 	const ref = useCopyToClipboard( text );
 	return (
-		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
-			variant="secondary"
-			ref={ ref }
-		>
+		<Button __next40pxDefaultSize variant="secondary" ref={ ref }>
```

Comment:
This can be seen when you have some unexpected error on editor and it breaks.

**Note:** It has some overridden style as height auto, so overall button size would get increased.

| Before | After     |
| :-------- | :------- |
| <img width="757" alt="error-boundry-before" src="https://github.com/user-attachments/assets/769791c6-3c2e-4cf1-8a70-13050bbe224a"> | <img width="894" alt="error-boundry-after" src="https://github.com/user-attachments/assets/dd5acf77-ddde-4dbd-bf2f-2f66a497363c"> |